packages/api/src/sequelize/migrations/2024-04-18_00_update-facility-organization.ts

```diff
@@ -0,0 +1,92 @@
+import { DataTypes } from "sequelize";
+import type { Migration } from "..";
+import { FacilityModel, FacilityType } from "../../models/medical/facility";
+import { OrganizationModel, OrganizationType } from "../../models/medical/organization";
+
+// Use 'Promise.all' when changes are independent of each other
+// Docs: https://sequelize.org/api/v6/class/src/dialects/abstract/query-interface.js~queryinterface
+export const up: Migration = async ({ context: queryInterface }) => {
+  return queryInterface.sequelize.transaction(async transaction => {
+    await queryInterface.addColumn(
+      FacilityModel.NAME,
```

Comment:
We should not make reference to external code that can evolve over time and mess with a snapshot of the DB on a particular point in time (that's what the migration is).

See recent migrations for reference.